etc/systemd/system/zfs-import-scan.service.in

```diff
@@ -9,4 +9,4 @@ ConditionPathExists=!@sysconfdir@/zfs/zpool.cache
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=@sbindir@/zpool import -d /dev/disk/by-id -aN
+ExecStart=/sbin/modprobe zfs && @sbindir@/zpool import -d /dev/disk/by-id -aN
```

Comment:
I would suggest adding a `-F` here to match the behavior when the module auto-imports the pool itself to fix #3434.